pkg/server/ServerData.go

```diff
@@ -0,0 +1,114 @@
+package server
+
+import (
+	"sync"
+
+	"github.com/solo-io/squash/pkg/models"
+)
+
+type ServerData struct {
+	debugAttachments        map[string]*models.DebugAttachment
+	debugAttachmentsMapLock sync.RWMutex
+	debugRequests           map[string]*models.DebugRequest
+	debugRequestsMapLock    sync.RWMutex
+}
+
+func NewServerData() *ServerData {
+	return &ServerData{
```

Comment:
Are these values initialized from the store?
Ideally you want a two way sync between the internal state and the store.